pyproject.toml

```
@@ -49,6 +49,7 @@ pyspark = { version = ">=3,<4", optional = true }
 requests = { version = ">=2,<3", optional = true }
 Shapely = { version = ">=1.6,<1.8.1", optional = true }
 sqlalchemy = { version = ">=1.3,<1.5", optional = true }
+duckdb = { version = "^0.3.2", optional = true }
```

```suggestion
duckdb = { version = ">=0.3.2,<1", optional = true }
```

This is so renovate (which isn't particularly smart about how it increases upper bounds) won't start adding `v1 || v2 || ... || vN` style version constraints.